Process.c

```diff
@@ -489,24 +489,21 @@ int Process_pidCompare(const void* v1, const void* v2) {
 }
 
 int Process_compare(const void* v1, const void* v2) {
-   const Process *p1, *p2;
-   const Settings *settings = ((const Process*)v1)->settings;
+   const Process *p1 = (const Process*)v1;
+   const Process *p2 = (const Process*)v2;
 
-   if (Settings_getActiveDirection(settings) == 1) {
-      p1 = (const Process*)v1;
-      p2 = (const Process*)v2;
-   } else {
-      p2 = (const Process*)v1;
-      p1 = (const Process*)v2;
-   }
+   const Settings *settings = p1->settings;
 
    ProcessField key = Settings_getActiveSortKey(settings);
 
    int result = Process_compareByKey(p1, p2, key);
 
+   if (Settings_getActiveDirection(settings) != 1)
```

Comment:
Maybe insert after the tie-breaker, cause `-0` should be `0`.

Reply:
That would make the tie breaker depend on sort order again which @BenBE objected against.

Reply:
@cgzones -0 == 0, thus this works as intended. Just reordered the code to make it remove several pointer casts that @fasterit introduced.

@fasterit I didn't object to anything. *g* Just put it in that order as there was some comment to avoid making the tie-breaker direction dependent for added stability of the sorting.